test/NuGet.Clients.Tests/NuGet.PackageManagement.UI.Test/Xamls/InfiniteScrollListTests.cs

```diff
@@ -19,7 +19,9 @@ public class InfiniteScrollListTests
     {
         public InfiniteScrollListTests()
         {
+#pragma warning disable VSSDK005 // Avoid instantiating JoinableTaskContext
             var joinableTaskContext = new JoinableTaskContext(Thread.CurrentThread, SynchronizationContext.Current);
+#pragma warning restore VSSDK005 // Avoid instantiating JoinableTaskContext
 
             NuGetUIThreadHelper.SetCustomJoinableTaskFactory(joinableTaskContext.Factory);
```

Comment:
Should we be disposing `joinableTaskContext` in `InfiniteScrollListTests.Dispose()` ?

Reply:
Sure, we can do that.